runtime/vscode/overrides/vscode-file-commands.vim

```diff
@@ -31,10 +31,8 @@ command! -bang -nargs=? Ex call <SID>editOrNew(<q-args>, <q-bang>)
 command! -bang Enew call <SID>editOrNew('__vscode_new__', <q-bang>)
 command! -bang Find call VSCodeNotify('workbench.action.quickOpen')
 
-command! -complete=file -bang -nargs=? Write if <q-bang> ==# '!' | call VSCodeNotify('workbench.action.files.saveAs') | else | call VSCodeNotify('workbench.action.files.save') | endif
 command! -bang Saveas call VSCodeNotify('workbench.action.files.saveAs')
 
-command! -bang Wall call VSCodeNotify('workbench.action.files.saveAll')
 command! -bang Quit if <q-bang> ==# '!' | call VSCodeNotify('workbench.action.revertAndCloseActiveEditor') | else | call VSCodeNotify('workbench.action.closeActiveEditor') | endif
 
 command! -bang Wq call <SID>saveAndClose()
```

Comment:
Can we remove these (saveAndClose/wq) too? Same with Quit? I think nvim can handle it now.

Reply:
"quit" may cause the nvim process to exit completely. Additionally, our layout management frequently deletes windows and buffers. Using altercommand for these operations generally poses no major issues. The commonly used command is write.